FRIEND/Settings.cpp

```diff
@@ -164,7 +166,7 @@ bool Settings::show()
 		}
 		
 		bool procEnabled = checkboxMask & 0x1;
-		bool docEnabled = checkboxMask & 0x2;
+		bool docEnabled = (checkboxMask >> 0x1) & 0x1;
```

Comment:
Actually, I was trying to make msvc happy with this I guess :) Your proposition is way better 👍